internal/file/file.go

```diff
@@ -0,0 +1,101 @@
+package file
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/afero"
+)
+
+const hashExt = ".sha256"
+
+// Copy copies a file from the given bytes to destination.
+func Copy(destination string, fin []byte) error {
+	if _, err := os.Stat(destination); err != nil && !os.IsNotExist(err) {
```

Comment:
Why do we need to `.Stat` here? The error will be returned from `.Create` if the file is not writable for example.

Reply:
I've removed file.Copy and replaced with os.WriteFile